weather_sp/README.md

```diff
@@ -26,14 +27,16 @@ _Common options_:
    using Python formatting, see [Output section](#output) below.
 * `-f, --force`: Force re-splitting of the pipeline. Turns of skipping of already split data.
 * `-d, --dry-run`: Test the input file matching and the output file scheme without splitting.
+* `--use-version`: Choose splitting implementation version (which depends on the data type)
```

Comment:
This only applies to grib though? I think that should be somewhere in the name, e.g. --grib-splitter-version seems clearer to me.

Reply:
Removing this flag based on your other suggestion.

Reply:
Should be removed from the README here as well? Or am I seeing an old version?

Reply:
Thanks! I missed this.